tests/integration/fuzzing/issue_590_test.cairo

```diff
@@ -0,0 +1,13 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import BitwiseBuiltin
+
+@external
+func test_safe_cast_fuzz{range_check_ptr, bitwise_ptr : BitwiseBuiltin*}(val : felt):
+    %{
+        if (ids.val % 2 == 0 and ids.val > (2**125 - 1) * 2):
+            expect_revert()
+    %}
+
+    return ()
```

Comment:
I don't understand this test case. Confusing name and magic numbers.

Reply:
I have renamed the test to match issue number, but I cannot help here more, because I also do not understand what is going on here and I would love to learn what codepaths are crashing here.